Demos/MaterialCreator/MaterialCreator.csproj

```diff
@@ -11,8 +11,8 @@
     <AssemblyName>MaterialCreator</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.16225.0</TargetPlatformVersion>
-    <TargetPlatformMinVersion>10.0.16225.0</TargetPlatformMinVersion>
+    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.16257.0</TargetPlatformVersion>
+    <TargetPlatformMinVersion>10.0.16257.0</TargetPlatformMinVersion>
```

Comment:
Confirmed, double checked and just ran it again using today's [public Insider Preview SDK release](https://www.microsoft.com/en-us/software-download/windowsinsiderpreviewSDK?wa=wsignin1.0) (16257). 

Thanks Danielle!